src/app/modules/time-entries/pages/time-entries.component.html

```diff
@@ -54,8 +54,14 @@
             <th class="col"></th>
           </tr>
         </thead>
+        <tr *ngIf="dataSource.isLoading">
+          <td class="text-center" colspan="7">No data available in table</td>
+        </tr>
         <app-loading-bar *ngIf="dataSource.isLoading"></app-loading-bar>
         <tbody *ngIf="!dataSource.isLoading">
+          <tr *ngIf="dataSource?.data.length === 0">
+            <td class="text-center" colspan="7">No data available in table</td>
```

Comment:
I think you can create a readonly variable to get this message, because you are repeating the same string

Reply:
Thank you for the review, I am going to apply this change.